cmd/main.go

```diff
@@ -63,10 +63,10 @@ func main() {
 		}
 		os.Exit(0)
 	}
-	klog.InfoS("Starting Azure Key Vault Provider", "version", version.BuildVersion)
+	klog.V(0).InfoS("Starting Azure Key Vault Provider", "version", version.BuildVersion)
```

Comment:
AFAIK, default log level is set to 0, so V(0) is redundant here. We can just revert to using `klog.InfoS` and not specify a level here.